scss/pages/blog-post.scss

```diff
@@ -0,0 +1,124 @@
+@import "common/base/topic-post";
+
+body.blog-post {
+  background-image: unset;
+  
+  li {
+    line-height: 1.6em;
+  }
+
+  blockquote {
+    border-left: 5px solid #e6e6e6;
```

Comment:
@MeghnaAJ If possible, see if you can use css color variables such as ``var(--primary)`` instead of hex codes.